### DISCUSSION TITLE
Timelord information missing

### DISCUSSION THREAD

**Discussion by number435398** [Chia-Network/chia-blockchain#7079] [Q&A]
I already posted this as a bug https://github.com/Chia-Network/chia-blockchain/issues/7077 but it was closed out after someone read the first sentence (referencing windows) and seemingly ignored the rest. So the issue remains: How do I start the timelord in linux (the exact command etc is missing in the documentation). What is the difference between a regular and bluebox timelord and what are the actual commands to start the bluebox timelord or make it a bluebox timelord instead of a regular timelord?

**Comment by joelrb** [comment]
Read https://github.com/Chia-Network/chia-blockchain/wiki/Timelords. You'll know the difference between regular and bluebox timelords.

**Reply by loppefaaret** [reply]
as for starting them, `chia start timelord`

**Reply by number435398** [reply]
> 
> 
> Read https://github.com/Chia-Network/chia-blockchain/wiki/Timelords. You'll know the difference between regular and bluebox timelords.

I realize now I did not sufficiently read the first paragraph previously.  But I still think it needs more explanation.  What motivation is there to run one versus the other?  What is the hardware difference needed? It seems like blueboxes are much more work intensive but don't seem to provide any return over a basic timelord, then again, based on what it says, it seems like a bluebox timelord would be disadvantageous to run instead of a regular timelord since it doesn't really act in the same way.

**Reply by loppefaaret** [reply]
you are right that we could explain bluebox timelords better than we have:
A bluebox timelord is different from a regular timelord, in that the regular timelord is working the tip of the chain, constantly engaging 3 tips, responsible for running the needed VDF's that all blocks need - a bluebox timelord can practically be anything, from a single core, to a 128 threadripper. what the bluebox timelord does, is getting feeded blocks from a node, that has uncompressed data in them(or not nearly as effeciently compressed as possible), and takes these blocks, and tries to compress them as much as possible, while also making it easy for a node to verify the data in it... neither kind of timelords are rewarded in any ways. 
so in short, a timelord is moving the chain, and the bluebox is optimizing it

**Comment by number435398** [comment]
But you guys claim that there is "value" in running a timelord, such as lower latency (presumably to decrease the chances of a lost reward).  But that appears to only apply to the regular timelord, while the bluebox appears to be more of a courtesy job or a thankless job as its function appears to have nothing to do with latency, etc, is that correct? Not to mention, you describe a bluebox timelord as basically anything, but what about the regular timelord?  What is the necessary computing power for a regular timelord?

**Reply by loppefaaret** [reply]
we do note that timelords are all doing the same work on 3 tips, so it is practically the fastest timelord that get it done first
but cause of the network component, there might be a slightly slower timelord that also finishes the job, and they are both propagating their work out over the network... the value is also in situation where netsplits and node outages happens, and momentarily, a local timelord could potentially save your win, by doing the work, and propagating the finished block, albeit a tad slower, but getting it done - once the split chains meet up again, if your chain is heavier ine weight, it will be chosen
